frontend/src/Components/Organisms/TagsTable/TagsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import { Post } from "_API/Types/Post";
+import { Tag } from "_API/Types/Tag";
+import Table from "_Components/Molecules/Table/Table";
+import { getTagData } from "_Services/SyncService";
+
+const TagsTable = () => {
+  const [tags, setTags] = useState<Tag[]>([]);
+  const [loaded, setLoaded] = useState(false);
+  
+  useEffect(() => {
+    const loadTagData = async () => {
+      let tagViewModels = await getTagData();
+      setTags(tagViewModels);
+    };
+    loadTagData();
```

Comment:
Once the arrow function is async we can await the `loadTagData` call, then call `setLoaded`